### PR TITLE
Do not allow us to automatically apply db DOWNs.

### PR DESCRIPTION
**Pull request by ndmckinley**
### Description
Like it says in the comment - don't auto apply database downs (so we can't dump the prod db anymore).

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: universal-application-tool-0.0.1/conf/application.conf [hunk 1]
```diff
@@ -306,9 +306,7 @@ play.evolutions {
   db.default {
     useLocks = true
     autoApply = true
-    # This might break something in the prod database someday.  Let's remove it once we completely agree
-    # never to require any more evolution downs.
-    autoApplyDowns = true
+    autoApplyDowns = false
```
Comment (hmhsu) [inline universal-application-tool-0.0.1/conf/application.conf:309]: Question (non-blocking): in what scenario would downs be applied when this is set to true? Could it happen when a server starts or does it only happen when the servers are shutdown?

**Review by hmhsu — APPROVED** [review]

File: universal-application-tool-0.0.1/conf/application.conf (hunk 1 above)
Comment (hmhsu) [reply universal-application-tool-0.0.1/conf/application.conf:309]: Read the doc. I see. the evolutions are applied when the scripts are changed.

**Review by bion — APPROVED** [review]

File: universal-application-tool-0.0.1/conf/application.conf (hunk 1 above)
Comment (hmhsu) [reply universal-application-tool-0.0.1/conf/application.conf:309]: I *think* some of the current ups are not idempotent though, like the ones that add a column.

Comment (ndmckinley) [reply universal-application-tool-0.0.1/conf/application.conf:309]: Hm, that's a good point - what's the equivalent to "if not exists" for adding a column.

Comment (ndmckinley) [reply universal-application-tool-0.0.1/conf/application.conf:309]: (It's not a big deal since we use evolution locks, but it would be nice to know!)

Comment (hmhsu) [reply universal-application-tool-0.0.1/conf/application.conf:309]: I *think* you can do

```sql
ALTER TABLE table_name ADD [COLUMN] IF NOT EXISTS column_name data_type;
```

Comment (ndmckinley) [reply universal-application-tool-0.0.1/conf/application.conf:309]: Ah, good eye!  I would do it to the old evolutions but as of this PR, we can't do that anymore, ha.

**Pull request merged**